torch/utils/data/_utils/collate.py

```diff
@@ -51,7 +51,7 @@ def default_collate(batch):
             # If we're in a background process, concatenate directly into a
             # shared memory tensor to avoid an extra copy
             numel = sum([x.numel() for x in batch])
-            storage = elem.storage()._new_shared(numel)
+            storage = elem.storage()._new_shared(numel * elem.element_size())
```

Comment:
Also wondering if we can kwarg this one, to help catch other sites where they are still passing numel